msg/sensor_accel_fifo.msg

```diff
@@ -8,8 +8,8 @@ float32 scale
 
 uint8 samples             # number of valid samples
 
-int16[32] x               # acceleration in the NED X board axis in m/s/s
-int16[32] y               # acceleration in the NED Y board axis in m/s/s
-int16[32] z               # acceleration in the NED Z board axis in m/s/s
+int16[32] x               # acceleration in the FRD board frame X-axis in m/s^2
```

Comment:
To add to the confusion these FIFO messages are the raw data that we get out of an IMU's FIFO. They're in the sensor frame and the data is raw 16 bit integers, typically with UINT16_MAX corresponding to the configured max range (eg 16 G accelerometer). That's why the message carries a rotation and scale factor.

Reply:
Ouch, so they're not always in the same frame as `sensor_accel` ?